explorer/templates/explorer/query_list.html

```diff
@@ -4,44 +4,46 @@
 {% block sql_explorer_content %}
     <div style="display: none">{% csrf_token %}</div>
     {% if recent_queries|length > 0 %}
-        <h3>
-            {% blocktrans trimmed with qlen=recent_queries|length %}
-                Your {{qlen}} Most Recently Run
-            {% endblocktrans %}
-        </h3>
-        <table class="table table-striped">
-            <thead>
-                <tr>
-                    <th>{% trans "Query" %}</th>
-                    <th>{% trans "Last Run" %}</th>
-                    <th class="text-center">CSV</th>
-                </tr>
-            </thead>
-            <tbody>
-                {% for object in recent_queries %}
+        <div class="container">
+            <h3>
+                {% blocktrans trimmed with qlen=recent_queries|length %}
+                    Your {{qlen}} Most Recently Run
+                {% endblocktrans %}
+            </h3>
+            <table class="table table-striped">
+                <thead>
                     <tr>
-                        <td class="name">
-                            <a href="{% url 'query_detail' object.query_id %}">{{ object.query.title }}</a>
-                        </td>
-                        <td>{{ object.run_at|date:"SHORT_DATETIME_FORMAT" }}</td>
-                        <td class="text-center">
-                            <a href="{% url 'download_query' object.query_id %}">
-                                <i class="glyphicon glyphicon-download"></i>
-                            </a>
-                        </td>
+                        <th>{% trans "Query" %}</th>
```

Comment:
As we'll be transitioning to 5.0 support soon and [probably] dropping 3.2 as it's EOL, it might be worth updating these to `translate` as I'm not entirely sure when the `trans` alias will be dropped.

Reply:
Opened as separate issue:
https://github.com/chrisclark/django-sql-explorer/issues/567